components/org.wso2.carbon.identity.api.server.application.management/org.wso2.carbon.identity.api.server.application.management.v1/src/main/resources/applications.yaml

```diff
@@ -2711,6 +2713,19 @@ components:
           description: >-
             Certificate value. If type is JWKS, value should be jwks URL. If
             type is PEM, value should be the certificate in PEM format.
+    ExternalizedConsentPage:
+      type: object
+      properties:
+        enabled:
+          type: boolean
+          description: >-
+            Decides whether external consent management is enabled.
+          default: false
+        consentPageUrl:
+          type: string
```

Comment:
is there a way to validate this to be a valid URL?

Reply:
Resolved from https://github.com/wso2/identity-api-server/pull/434/commits/0180146d25d577568965efdc48757dd43680f63e